internal/client/hub/hub.go

```diff
@@ -865,7 +865,7 @@ func UpdateExternalAPI(externalApiID string, displayName string, description str
 	q.Set("updateMask", strings.Join(updateMask, ","))
 	u.RawQuery = q.Encode()
 
-	payload, err := getExternalApi(displayName, description, endpoints, paths, externalUri)
+	payload, err := getExternalApi(displayName, description, endpoints, paths, externalUri, apiType)
```

Comment:
same comment as above

Reply:
No, this is correct. It is not attribute but `apiType`.